dev/Expander/Expander.xaml

```diff
@@ -88,37 +163,21 @@
                             MaxWidth="{TemplateBinding MaxWidth}"
                             contract7Present:CornerRadius="{TemplateBinding CornerRadius}"
                             IsEnabled="{TemplateBinding IsEnabled}"
+                            Padding="{TemplateBinding Padding}"
                             Style="{StaticResource ExpanderHeaderDownStyle}"
                             Content="{TemplateBinding Header}"
                             ContentTemplate="{TemplateBinding HeaderTemplate}"
                             ContentTemplateSelector="{TemplateBinding HeaderTemplateSelector}"
                             HorizontalAlignment="{TemplateBinding HorizontalAlignment}"
-                            IsChecked="{Binding Path=IsExpanded, Mode=TwoWay, RelativeSource={RelativeSource TemplatedParent}}"/>
-
-                        <Border
-                            x:Name="ExpanderContent"
-                            Grid.Row="1"
-                            Visibility="Collapsed"
-                            Background="{ThemeResource ExpanderDropDownBackground}"
-                            contract7Present:BackgroundSizing="{TemplateBinding BackgroundSizing}"
-                            BorderBrush="{ThemeResource ExpanderDropDownBrush}"
-                            BorderThickness="{TemplateBinding BorderThickness}"
-                            MinHeight="{TemplateBinding MinHeight}"
-                            MinWidth="{TemplateBinding MinWidth}"
-                            MaxWidth="{TemplateBinding MaxWidth}"
-                            contract7Present:CornerRadius="{TemplateBinding CornerRadius}"
-                            Padding="{TemplateBinding Padding}">
-                            <ContentPresenter
-                                Content="{TemplateBinding Content}"
-                                ContentTemplate="{TemplateBinding ContentTemplate}"
-                                ContentTemplateSelector="{TemplateBinding ContentTemplateSelector}"
-                                HorizontalContentAlignment="{TemplateBinding HorizontalContentAlignment}"
-                                VerticalContentAlignment="{TemplateBinding VerticalContentAlignment}" />
-                        </Border>
+                            IsChecked="{Binding Path=IsExpanded, Mode=TwoWay, RelativeSource={RelativeSource TemplatedParent}}" />
                     </Grid>
                 </ControlTemplate>
             </Setter.Value>
         </Setter>
     </Style>
 
+    <Style x:Key="ExpanderHeaderUpStyle" TargetType="ToggleButton" BasedOn="{StaticResource ExpanderHeaderDownStyle}">
+        <Setter Property="Tag" Value="{StaticResource ExpanderChevronUpGlyph}" />
```

Comment:
>Property [](start = 16, length = 8)

How is this used? Needing to use the tag property usually indicates something is wrong..

Reply:
Exactly the same style but one has chevron up and one has chevron down, but I can't pass it as content since the content can be anything else in the header - it's just the chevron that's different. The other way to do it is to compltely copy the header style just because of the different chevron :/